tests/integration/test_k8s_with_cluster.py

```diff
@@ -46,28 +45,25 @@
 
 @mark.usefixtures("setup_cluster")
```

Comment:
Why have you gone down the `@mark.usefixtures("setup_cluster")` route for fixture, rather than how we've been doing them up to this time?

This is for my own understanding.

Reply:
Because we don't need access to the fixture object as there is none therefore we want to avoid the requirement of passing a parameter to the test method.

Reply:
I thought that might be the case - thanks.